managed/services/telemetry/config.default.yml

```diff
@@ -761,47 +761,47 @@ telemetry:
   #Grafana
   - id: GrafanaUsersCount
     source: GRAFANADB_SELECT
-    query: count(*) AS count FROM user
+    query: count(*) AS count FROM public.user
```

Comment:
Why did you specify schema? Grafana uses multiple schemas in their database?

Reply:
`user` is an internal function in PostgreSQL (and a reserved word) - it returns the current db user. 

While this query will run OK - `SELECT * FROM user;`, and even this will run too - `SELECT user;`, this one - `SELECT id FROM user` - will fail with `ERROR: column "id" does not exist` even though the column `id` does exist.

Therefore, to prevent those `user` table queries from failing, we have to prefix them with their schema.

Reply:
Aha, I got what is the issue here. `user` and `public.user` are two different things, first one in reserved word for current PG user and second one is ordinary grafana table with grafana users.

So all these queries were invalid until your changes. Good 👍